.github/workflows/build.yml

```diff
@@ -111,7 +111,6 @@ jobs:
             brew list --cask | xargs brew uninstall --force --ignore-dependencies
             brew update
             brew bundle
-            brew link mbedtls@2 --force # needed for CMake
```

Comment:
Does this mean this is no longer needed, or do we simply not care about mbedtls@2 anymore?

Reply:
Now that mbedtls 3.x is supported HL can link with the normal mbedtls package, which does does not require force linking to be found by CMake.